### PR TITLE
Adds character functionality to the game

### LINKED ISSUE #68: Allow choosing game characters
### Expected results

Before the game starts, we want to choose different characters to play with.
This is useful for the [GCI task][task].

### Actual results

Currently, we do not have any characters we can choose.

### Hints to solve

- [x] create a character
- [x] add the character to a folder
- [ ] link that folder in the task
- [ ] add a javascript file which loads characters from a list of characters
- [x] show the character on the tile it walks on

There is no need to do all of this at once.
Once we completed that, we can open up the GCI task for new contributors to the labyrinth.

[task]: https://github.com/fossasia/labyrinth/commit/abc31b31513f9b7e95ab46a1bb9203c86931fd14

### PR DESCRIPTION
**Pull request by abishekvashok**
This pull request is related to issue #68 

The problem I want to solve is: to faciliate adding a character

My steps to solve it:
Changes
Added a character, defined some objects for the player class and made movements possible.
Adds a basic character (currently a robot) to the game. Modifies needy
files and adds new objects to make the character move from tile to tile.

Screenshots/Links:[Deploy/Preview Links: http://rawgit.com/abishekvashok/labyrinth/master/index.html ]
screenshot_20180105_173432
screenshot_20180105_173439

Fixes #68

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
http://rawgit.com/abishekvashok/labyrinthlabyrinth/index.html

**Comment by abishekvashok** [comment]
@YashKumarVerma sry updated it

**Comment by abishekvashok** [comment]
@YashKumarVerma pls check now
screenshot_20180105_174311

File: js/tiles.js [hunk 1]
```diff
@@ -59,6 +59,8 @@ ImageCollection.prototype.showIn = function(container) {
 };
 ImageCollection.prototype.playerEnters = function(player) {
   this.container.classList.add("current");
+  /* 214.772 is half of the tile width while 128.157 is half the height of the tile */
+  player.setPosition(this.pixelPosition.x+214.772, this.pixelPosition.y+128.157);
```
Comment (niccokunzmann) [inline js/tiles.js]: please do not use hard-coded values. There are variables for that which you should use.
For better understanding, you can add a method getCenterPixelPosition() and pass that to setPosition.

And I think, if you rename `setPosition` to `displayAtPosition`, this should be more descriptive.

Comment (niccokunzmann) [reply js/tiles.js]: Calling the player back is nice. I like it. :)

Comment (abishekvashok) [reply js/tiles.js]: sure

**Comment by niccokunzmann** [comment]
Thanks for working on that :)

**Comment by abishekvashok** [comment]
Btw why did you go for a merge? @niccokunzmann should I make a new PR now?

**Comment by niccokunzmann** [comment]
Because it works and if I merge later, you may have conflicts and not others have it easier to work on this and in case you are absent, we can improve it.

**Comment by niccokunzmann** [comment]
Yes, you can make a new pr.

**Comment by abishekvashok** [comment]
oh thanks then @niccokunzmann :)

**Pull request merged**